docs/start/setup-channel.mdx

```diff
@@ -5,7 +5,7 @@ description: Installation and setup instructions for IBM Quantum Platform or IBM
 ---
 # Select and set up an IBM Quantum channel
 
-You can access IBM Quantum systems by using the IBM Quantum Platform or IBM Cloud channel.
+You can access IBM Quantum systems by using the IBM Quantum Platform or IBM Cloud _channel_.  _Channel_ is the term used to describe the method you use to access IBM Quantum systems. 
```

Comment:
nit - I feel it's better to say IBM Quantum _services_ instead of systems, since we provide more than just hardware access.